src/main/kotlin/com/pubnub/api/endpoints/FetchMessages.kt

```diff
@@ -107,5 +131,10 @@ class FetchMessages internal constructor(
         page.end?.run { queryParams["end"] = this.toString().lowercase(Locale.US) }
 
         if (includeMeta) queryParams["include_meta"] = includeMeta.toString()
```

Comment:
In this case:
includeMeta.toString()
includeMessageType.toString()
includeSpaceId.toString()
are always true.

What do you think about changing those values to "true" ?

Reply:
Sure. I'll do it